packages/cascara/src/lib/mock/generateFakeEmployees.js

```diff
@@ -6,15 +6,18 @@ export const generateFakeEmployees = (qty) =>
   new Array(qty).fill(null).map((e, i) => {
     // So we never have collisions, we have to start at 2 instead of 1
     faker.seed(i++ + 1);
+
+    const firstName = faker.name.firstName();
+    const lastName = faker.name.lastName();
+
     return (e = {
```

Comment:
What we want with this fixture data is to have one field of each supported type.